### DISCUSSION TITLE
Install Wizard not running?

### DISCUSSION THREAD

**Discussion by iTracMan** [magicbug/Cloudlog#2544] [General]
Hi fellow users, I am after much learning, within a hair's breath of getting CloudLog running on a pi-4 but I have fallen at the last hurdle.

Ubuntu 22.04.06, Apache2 2.4.52, MySQL 8.0.34, PHP 8.1.2 all installed and running. Cloudlog unpacked into /var/www/html/cloudlog and appropriate paths and permissions set as per the wiki.

The attached screen loads but when I hit Install it just goes to a blank screen and no installation takes place. 

Could anyone guide me on where I may have gone wrong please?

![Screenshot](https://github.com/magicbug/Cloudlog/assets/29685970/c9019807-93a2-408b-9a60-318db0fdfa8f)

**Comment by iTracMan** [comment]
No idea how or why but I went back and deleted the img on the Pi4, started again from scratch and this time it worked?

**Comment by magicbug** [comment]
Strange but pleased its resolved

**Comment by iTracMan** [comment]
Thanks Peter,

I guess it’s the linux equivalent  of Ctrl Alt Delete :-)
